LedgerSMB/Scripts/contact.pm

```diff
@@ -340,7 +341,7 @@ sub _main_screen {
                    person => $person,
                  employee => $person,
                      user => $user,
-                    roles => eval { $user->list_roles },
+                    roles => \@roles,
```

Comment:
I suspect this isn't a syntax error.  It is rather a syntax/semantics mismatch.  Thanks for fixing!